src/main/java/org/gridsuite/filter/expertfilter/expertrule/PropertiesExpertRule.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.filter.expertfilter.expertrule;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.iidm.network.Identifiable;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+import org.gridsuite.filter.FilterLoader;
+import org.gridsuite.filter.identifierlistfilter.FilterEquipments;
+import org.gridsuite.filter.utils.expertfilter.DataType;
+
+import java.util.*;
+
+import static org.gridsuite.filter.utils.expertfilter.ExpertFilterUtils.getFieldValue;
+
+/**
+ * @author maissa SOUISSI <maissa.souissi at rte-france.com>
+ */
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@SuperBuilder
+public class PropertiesExpertRule extends AbstractExpertRule {
```

Comment:
In my opinion, PropertiesExpertRule can extend StringExpertRule, just adding propertyName

Reply:
done